SwiftWisdom/Core/StandardLibrary/Numbers/BinaryFloatingPoint+Extensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  BinaryFloatingPoint+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Stephen Wong on 5/1/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension BinaryFloatingPoint {
+    /// https://codereview.stackexchange.com/questions/142748/swift-floatingpoint-rounded-to-places
```

Comment:
We might want to give this link the `- SeeAlso:` tag in the documentation along with adding a basic summary of the function.